src/authority.rs

```diff
@@ -61,6 +61,17 @@ impl Authority {
         }
     }
 
+    /// Returns true if a client, false if a node or group.
+    pub fn is_client(&self) -> bool {
+        match *self {
+            Authority::ClientManager(_) |
+            Authority::NaeManager(_) |
+            Authority::NodeManager(_) |
+            Authority::ManagedNode(_) => false,
+            Authority::Client { .. } => true,
+        }
+    }
+
```

Comment:
I personally find `match` more readable (and easier to modify in the future if we add some other client type), but I won't argue ;)